src/HPND-export2-US.xml

```diff
@@ -0,0 +1,42 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<SPDXLicenseCollection xmlns="http://www.spdx.org/license">
+   <license isOsiApproved="false" licenseId="HPND-export2-US" name="HPND with
+   US Government export control and 2 disclaimers" listVersionAdded="3.24">

```

Comment:
Line break in value of attribute. Please move it to a single line.

Reply:
Resolved in bf1a87f, thanks @xsuchy!